src/Swashbuckle.AspNetCore.SwaggerGen/Annotations/XmlCommentsOperationFilter.cs

```diff
@@ -80,9 +80,10 @@ private void ApplyResponsesXmlToResponses(IDictionary<string, Response> response
             while (responseNodes.MoveNext())
             {
                 var code = responseNodes.Current.GetAttribute("code", "");
-                var response = responses.ContainsKey(code)
-                    ? responses[code]
-                    : responses[code] = new Response();
+
+                Response response;
+                if (!responses.TryGetValue(code, out response))
+                    responses[code] = response = new Response();
```

Comment:
These changes are already in a separate PR. I really appreciate the changes but I have a strong preference to restrict each PR to a specific concern. E.g. - if this one is about adding the XML comments, then I'd prefer to not pollute it with other changes